assets/sass/protocol/components/_button.scss

```diff
@@ -2,8 +2,6 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-/* stylelint-disable no-duplicate-selectors */
```

Comment:
Comment (non-blocking): FWIW, I think it's probably OK to have rules like this at the top of a file, if the whole file relies upon it. My nit in the prior review was that it shouldn't come on line 1, above the license block, as that makes it hard to spot since eyes can misinterpret it as a singular comment.

But no need to change it again as what you have here still works :)